source/common/quic/platform/quic_mem_slice_span_impl.h

```diff
@@ -55,22 +60,29 @@ class QuicMemSliceSpanImpl {
 
 private:
   Envoy::Buffer::Instance* buffer_{nullptr};
+  // Nullptr if this span is not constructed with a QuicMemSlice.
+  QuicMemSliceImpl* mem_slice_{nullptr};
```

Comment:
Can you add some comments to explain the relationship of buffer_ and mem_slice_?  (It seems we only need one of them to be !nullptr at a time, which is easier to understand)

Reply:
done. buffer_ always points to the actually memory, but mem_slice_ is only used in the constructor which takes QuicMemSlice.